src/renderer/components/ManagerConnect/index.js

```diff
@@ -292,6 +298,11 @@ const ManagerConnect = ({
     );
   }
 
+  if (onSuccess) {
+    onSuccess({ device, deviceInfo, result });
+    return null;
+  }
```

Comment:
ok, i apologize that this was not existing yet. unfortunately the current implementation is going to create bugs (could double call onSuccess. not a good idea to call in render step). please look and try to get in this implementation instead: https://github.com/LedgerHQ/lld-v2/blob/poc-device-connect/src/renderer/components/DeviceAction/index.js#L195

**OR maybe we should just get this in the PR #127**